vehicles/Steering/chassis.cc

```diff
@@ -206,14 +207,60 @@ void chassisTask(void* arg) {
   chassis->SteerThetaReset();
   chassis->SetWheelSpeed(0,0,0,0);
 
+  float v_max = 660;
+  float normalization_factor = 1;
+#ifdef LIMITED_SAMPLE_MATCH
+  v_max *= 0.5;
+#endif
+  // Prevent vehicle from flipping due to sudden changes in velocity.
+  // Speed at which safety will not kick in
+  // TODO: Find good numbers
+  float momentum_safe_speed = v_max * 0.4; // Magic number
+  float momentum_safe_factor_line = 0.3;   // also ^
+  float momentum_safe_factor_turn = 0.3;   // also ^
+  float momentum_factor = 0.75;            // also ^
```

Comment:
Please use uppercase for const name. Also, consider using `constexpr float` or `const float`